src/rf/requirements/workers.txt

```diff
@@ -1,2 +1,3 @@
 GDAL==1.10.0
 numpy==1.10.1
+exifread==2.1.2
```

Comment:
Actually it turns out we already set it up so that workers and app servers get the same requirements.
https://github.com/azavea/raster-foundry/blob/develop/deployment/ansible/roles/raster-foundry.app/tasks/dependencies.yml#L18
I think you just need to re-provision app and worker to get the new packages.